docs/source/getting-started/app-architecture.rst

```diff
@@ -0,0 +1,108 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+App Architecture
+****************
+
+In this section we'll look at the different components of our social network app.
+The goal is to familiarise you enough to feel comfortable extending the code with a new feature in the next section.
+
+There are two main components in the code - the DAML model and the React/TypeScript frontend - with generated TypeScript code to bridge the two.
+Let's start by looking at the DAML model, as this sets the core logic of the application.
+
+The DAML Model
+==============
+
+Using VSCode (or a code editor of your choice), navigate to the ``daml`` subdirectory.
+There is a single DAML file called ``User.daml`` with the model for users of the app.
```

Comment:
"model for users of the app" is slightly ambiguous. We need something better but I have currently no idea what.